app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -1533,6 +1533,10 @@ private void handleWordPair() {
 
             case "time":
                 Log.d(TAG, "processing time keyword");
+                // Add the dot to the time if it's missing.
+                if (thisnumber >= 100)
```

Comment:
Can this be moved to `PhoneKeyPadInputActivity` so that we don't have to worry about changes in behavior with the voice input?

Reply:
I thought it would be nice to be able to optionally voice input the time as "twelve hundred time".  Let me know what you think.  I can move it if you want.